src/index.js

```diff
@@ -67,7 +67,7 @@ function nowApi () {
         if (r.state === 'READY') {
           return r
         }
-        if (r.state === 'DEPLOYING' || r.state === 'BOOTED') {
+        if (~['DEPLOYING', 'BOOTED', 'BUILDING'].indexOf(r.state)) {
```

Comment:
I *hate* using `~` to check `-1` - can you rewrite the condition to be clear in this case?